config/config.go

```diff
@@ -417,6 +401,18 @@ func (s *SourceBlockchain) Validate(destinationBlockchainIDs *set.Set[string]) e
 		}
 	}
 
+	// Validate and store the subnet and blockchain IDs for future use
```

Comment:
Not for this PR, but we should consider splitting `config.go` up into multiple files at some point soon. I think `SourceBlockchain` and `DestinationBlockchain` could each be put in their own file still in the `config` package and could help improve navigation/readability.

Reply:
agreed and some of the similarities between the source/destination can be abstracted

Reply:
> some of the similarities between the source/destination can be abstracted

Viper doesn't play nicely with this unfortunately. There may be a workaround, but I haven't looked to deeply into it.